addons/mail/static/src/js/utils.js

```diff
@@ -5,6 +5,14 @@ var core = require('web.core');
 
 var _t = core._t;
 
+/**
+ * WARNING: this is not enough to unescape potential XSS contained in htmlString, transformFunction
+ * should handle it or it should be handled after/before calling parseAndTransform. So if the result
+ * of this function is used in a t-raw, be very careful.
+ * @param {string} htmlString
```

Comment:
good enough, the most important point was to draw attention to this issue and that's the case now.

add new line after comment